api/src/neo4j/queries/search.js

```diff
@@ -277,10 +277,14 @@ const modelSearch = async ({ searchTerm, model, version, limit }) => {
 const _search = async ({ searchTerm, model, version, limit, includeCounts }) => {
   const v = version ? `:V${version}` : '';
 
+  // the EC field for reaction could contain ":", which is a special character
+  // in this case th search term is modified to be escape and perform an exact match
+  const term = searchTerm.includes("EC:") ? `\\"${searchTerm}~\\"` : `${searchTerm}~`;
```

Comment:
`includes` and not `startsWith`?

Reply:
Might be more accurate but I wouldn't bother changing it unless you think any other field can contain this substring.

Reply:
`EC` on its own could be present in other strings, but with `:` it's extremely unlikely.